packages/cms-cli/bin/cli.js

```diff
@@ -35,9 +34,16 @@ const argv = yargs
   .usage('Tools for working with the HubSpot CMS')
   .middleware([setLogLevel])
   .exitProcess(false)
-  .fail((msg, err /*, _yargs*/) => {
+  .fail((msg, err, _yargs) => {
+    // Preserve stack trace.
+    if (err) throw err;
+
     if (msg) logger.error(msg);
     if (err) logErrorInstance(err);
+
+    // Give command-specifc help
+    console.log(_yargs.help());
+    process.exit(0);
```

Comment:
I think we should be using `process.exit(1)` so that scripts are properly notified of the error.

Reply:
Ok will do.  I had done 0 since we already give error reasons at the top, but that makes sense for scripts